### PR TITLE
Make sure we only pull reviewed translations from transifex

### PR DESCRIPTION
**Pull request by amangano-edx**
@rlucioni @andy-armstrong

### CODE REVIEW

File: {{cookiecutter.repo_name}}/Makefile [hunk 1]
```diff
@@ -67,7 +67,7 @@ compile_translations:
 fake_translations: extract_translations dummy_translations compile_translations
 
 pull_translations:
-	tx pull -a
```
Comment (amangano-edx) [inline {{cookiecutter.repo_name}}/Makefile:70]: Is there any reason why we shouldn't use the force flag (f) here, as we do in https://github.com/edx/cookiecutter-django-app/pull/18/files#diff-af4c0c482ca0bcd7910328df7120974dL88 ?

Comment (rlucioni) [reply {{cookiecutter.repo_name}}/Makefile:70]: @amangano-edx no, please add the force flag. I explained why it's necessary in https://github.com/edx/course-discovery/pull/457.

**Review by rlucioni — APPROVED** [review]
LGTM once the force flag is added.

**Review by andy-armstrong — APPROVED** [review]
👍

**Comment by amangano-edx** [comment]
@rlucioni I updated the module paths for django_social, per your request.

**Review by rlucioni — APPROVED** [review]
@amangano-edx looks good.

**Pull request merged**